taskipy/exceptions.py

```diff
@@ -32,6 +32,15 @@ def __init__(self, task_name: str):
     def __str__(self):
         return f'could not find task "{self.task}"'
 
+class MalformedTaskError(TaskipyError):
+    def __init__(self, task_name: str, reason: str):
+        super().__init__()
+        self.task = task_name
```

Comment:
Do we want these to be "private"?

Reply:
I think not, as this data can be used for programmatic introspection if needed. Anyways, I was following your code when you implemented `TaskNotFoundError` ;)